angular-app/src/app/workspace/workspace.component.css

```diff
@@ -48,7 +53,7 @@ mat-form-field {
     font-weight: 400;
 }
 
-.save-button {
+.right-button {
```

Comment:
The CSS you have here is generic enough, you could just call this class `.right` and use on on things outside of buttons.